locale/en/docs/guides/backpressuring-in-streams.md

```diff
@@ -42,11 +42,10 @@ const readline = require('readline');
 // process.stdin and process.stdout are both instances of Streams
 const rl = readline.createInterface({
   input: process.stdin,
-  output: process.stdout
+  output: process.stdout,
```

Comment:
Feel free to do that, if you think we should use lax commas keep it and maybe wait for other feedbacks?